core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -510,7 +510,7 @@ public enum BuiltInMethod {
   SAFE_MULTIPLY(SqlFunctions.class, "safeMultiply", double.class, double.class),
   SAFE_SUBTRACT(SqlFunctions.class, "safeSubtract", double.class, double.class),
   LOG(SqlFunctions.class, "log", long.class, long.class),
-  LOG2(SqlFunctions.class, "log2", long.class),
+  LOGMS(SqlFunctions.class, "logMS", long.class, long.class),
```

Comment:
Why delete `LOG2` here, but `SqlLibraryOperators` has no deletion

Reply:
The log2 function was originally designed to adapt to mysql and spark functions. This function adapts to the behavior of all log functions in mysql and spark. I made some reasonable reconstructions. In order to improve the readability and maintainability of the code, you can Found that the log2 function is still effective